bravado/swagger_model.py

```diff
@@ -40,6 +40,9 @@ def wait(self, timeout=None):
         with contextlib.closing(urllib.urlopen(self.get_path())) as fp:
             return self.FileResponse(json.load(fp))
 
+    def result(self, *args, **kwargs):
+        return self.wait(*args, **kwargs)
```

Comment:
This is because `FileEventual` [could also be a result](https://github.com/Yelp/swagger-py/blob/swagger2.0/bravado/swagger_model.py#L54-55) of `start_request`, so had to add `result()` to its interface as well.